src/esp/assets/attributes/ObjectAttributes.h

```diff
@@ -18,6 +20,14 @@ namespace attributes {
  */
 class AbstractObjectAttributes : public AbstractAttributes {
  public:
+  /**
+   * @brief This defines an example JSON descriptor for @ref
+   * AbstractObjectAttributes Has values that are different than defaults so
+   * this can be used to test JSON loading. These values may break the code, so
+   * should not be used.
```

Comment:
```suggestion
   * this can be used to test JSON loading. These values may be purposefully invalid for testing, so
   * should not be used in practice.
```
Small suggestion on wording this warning. Same for others.